src/concrete/ml/sklearn/qnn_module.py

```diff
@@ -27,13 +28,15 @@ def __init__(
         n_layers: int,
         n_outputs: int,
         n_hidden_neurons_multiplier: int = 4,
-        n_w_bits: int = 3,
-        n_a_bits: int = 3,
-        n_accum_bits: int = MAX_BITWIDTH_BACKWARD_COMPATIBLE,
+        n_w_bits: int = 4,
+        n_a_bits: int = 4,
+        # No pruning by default as roundPBS keeps the PBS precision low
```

Comment:
What' is the chosen bit width to round down?

Reply:
So these hyper parameters are useless? I am trying to understand what is the number of bits that's used to round?